index.js

```diff
@@ -2,17 +2,39 @@
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
-
 /***** Deliverable 2 *****/
+header.style.color = "cornflowerblue"
 
+/***** Deliverable 3 *****/
+console.log('This is what the traveler object looks like: ', traveler)
 
+const travelerName = document.querySelector("div h2")
+travelerName.innerHTML = traveler.name
 
-/***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+const travelerNickname = document.querySelector("div em")
+travelerNickname.innerHTML = traveler.nickname
```

Comment:
Good! One recommendation I have is to choose ```textContent``` over ```innerHTML``` when possible especially if you know what you're inserting should be plain text. It's better practice for security reasons. You can read more about the security implications here: https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#security_considerations